xbmc/utils/GroupUtils.cpp

```diff
@@ -100,16 +100,17 @@ bool GroupUtils::Group(GroupBy groupBy, const std::string &baseDir, const CFileI
       setInfo->m_strTitle = pItem->GetLabel();
 
       int ratings = 0;
+      float tot = 0;
```

Comment:
Please don't abbreviate your variable names to much. It makes the code less obvious.